mkdocs.yml

```diff
@@ -50,12 +37,25 @@ nav:
           - AMI Instructions and Guides:
             - Spreadsheet Formatting Overview: ami_spreadsheet_overview.md
             - Ingesting New Digital Objects and Collections using Spreadsheets or Google Sheets: AMIviaSpreadsheets.md
-            - Configuration for Google Sheets API: googleapi.md
+            - Linked Data Reconciliation: ami_lod_rec.md
             - Using the Islandora 7 Solr Importer: I7solrImporter.md
+            - Configuration for Google Sheets API: googleapi.md	  
+        - Annotations: annotations.md
+    - Site Administration & Configuration:
+      - Strawberryfield Formatters: strawberryfield-formatters.md
+      - Creating Display Modes: createdisplaymodes.md
+      - Archipelago's File Persistence Strategy: archifilepersistencestrategy.md
+      - How to Set Up SSL for Docker/Archipelago: sslsetup.md
+      - Debugging PHP in Archipelago: xdebug.md
+      - General Q&A:
+        - generalqa_index.md
+            - Twig Modules Configuration: generalqa_twig_modules_configuration.md
+            - SMTP Configuration: generalqa_smtp_configuration.md
+            - Min.io Logging: generalqa_minio_logging.md
```

Comment:
```suggestion
        - generalqa_index.md
            - Twig Modules Configuration: generalqa_twig_modules_configuration.md
            - SMTP Configuration: generalqa_smtp_configuration.md
            - Min.io Logging: generalqa_minio_logging.md
```
The spacing here was a little funky and will cause an error when run locally. Not sure if this results in what you had in mind, but we can adjust further if not.

Reply:
Ok, yes, wanted to keep as-is from the current mkdocs.yml. The spacing issues are very odd--as they are *not* present in the copy I have on my local.

![Screen Shot 2022-08-09 at 10 31 33 AM](https://user-images.githubusercontent.com/67483756/183676001-2200ec63-b561-4945-b01f-a3b405a30db1.png)